### ISSUE TITLE
Missing Accessibility Features

### ISSUE THREAD

**Issue by PasanSWijekoon** [PasanSWijekoon/UMeeChat-App-FrontEnd#18]
Some of the UI components lack accessibility features (e.g., missing accessibilityLabel for image picker).
